charts/grafana/values.yaml

```
@@ -194,6 +194,7 @@ serviceMonitor:
   tlsConfig: {}
   scrapeTimeout: 30s
   relabelings: []
+  targetLabels: []
```

This should be empty map `{}`:
```suggestion
  targetLabels: {}
```

---

Why? `targetLabels` is a list.

---

Then, did you provide an example in values.yaml? How does an engineer / developer know?

---

In the same way, how he knows set `relabelings`. I can provide list of string as example. Does it help?

```
% k explain servicemonitor.spec.targetLabels
KIND:     ServiceMonitor
VERSION:  monitoring.coreos.com/v1

FIELD:    targetLabels <[]string>

DESCRIPTION:
     TargetLabels transfers labels from the Kubernetes `Service` onto the
     created metrics.
```

---

Yes.